Kommunicate/Classes/Views/AwayMessageView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  AwayMessageView.swift
+//  Kommunicate
+//
+//  Created by Mukesh on 08/01/19.
+//
+
+import Foundation
+import ApplozicSwift
+
+class AwayMessageView: UIView {
+
+    struct Padding {
+        struct dottedLineView {
+            static let leading: CGFloat = 12.0
+            static let trailing: CGFloat = 23.0
+        }
+
+        struct messageLabel {
+            static let top: CGFloat = 5.0
+            static let leading: CGFloat  = 20.0
+            static let trailing: CGFloat = 20.0
+        }
+    }
+
+    private let messageLabel: UILabel = {
+        let label = UILabel(frame: CGRect.zero)
+        label.font = UIFont(name: "HelveticaNeue-Light", size: 14)
+        label.contentMode = .center
+        label.textAlignment = .center
+        label.textColor = UIColor(netHex: 0x676262)
+        label.numberOfLines = 4
+        return label
+    }()
+
+    private let dottedLineView: UIView = {
+        return UIView(frame: CGRect.zero)
+    }()
+
+    private let dottedLayer: CAShapeLayer = {
+        let shapeLayer = CAShapeLayer()
+        shapeLayer.strokeColor =  UIColor(netHex: 0xbebbbb).cgColor
+        shapeLayer.lineWidth = 1
+        shapeLayer.lineDashPattern = [5, 5]
+        shapeLayer.path = CGMutablePath()
+        return shapeLayer
+    }()
+    private let dottedLineViewHeight: CGFloat = 1.0
+    lazy private var dottedLineHeightAnchor = dottedLineView.heightAnchor.constraint(equalToConstant: dottedLineViewHeight)
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    func commonInit() {
```

Comment:
We can change this name to setupConstraints() or setupViews() since we are only setting constraints here.